maskit/examples/load_mnist.py

```diff
@@ -0,0 +1,122 @@
+import tensorflow as tf
+import collections
+from sklearn.decomposition import PCA
+from pennylane import numpy as np
+from sklearn.preprocessing import minmax_scale
+
+np.random.seed(42)
+
+MAX_TRAIN_SAMPLES = 11471
+MAX_TEST_SAMPLES = 1952
+
+
+def reduce_image(x):
+    x = np.reshape(x, [1, 28, 28, 1])
+    x = tf.image.resize(x, [4, 4])
+    x = np.reshape(x, [4, 4])
```

Comment:
Is the reshape needed here? It looks like ``tf.image.resize`` already returns this shape.

Reply:
The resize function returns 1x4x4x1 as far as I can tell, so it requires another reshape.